bin/phpliteadmin

```diff
@@ -14,24 +14,20 @@ if [[ ! -r "$1" ]]; then
     exit 1
 fi
 
-if [ $IDE_OFFLINE ]; then
+if [ "${IDE_OFFLINE+x}" ]; then
```

Comment:
This just expands to `"x"` only if `$IDE_OFFLINE` is set (even if null). I think this is just a way to distinguish between whether a variable is set or not. I don't think it's strictly necessary here, but Dan A used it in other scripts, so just wanted to stay consistent for now.

@glennholloway do you see any value in keeping this?